user-docs/pupil-capture.md

```diff
@@ -158,11 +161,11 @@ The green outline show the calibrated area. Orange shows the sampled data points
 
 ### Pupil Detection
 
-Adjust the headset according to your comfort and to ensure a good calibration result
+The eye camera constantly record and track the subject's eye movements to build a 3d eye model. To ensure a good detection, roll your eyes around while keeping the detection tracking on the pupil.
```

Comment:
Pupil's algorithms automatically detect the participant's pupil. With the 3d detection and mapping mode, Pupil uses a 3d model of the eye(s) that constantly updates based on observations of the eye. This enables the system to compensate for movements of the headset - slippage. To build up an initial model, you can just look around your field of view.